include/crm/common/xml_internal.h

```diff
@@ -447,4 +447,7 @@ xmlNode *pcmk__xml_create_patchset(const xmlNode *before, xmlNode *after,
 int pcmk__xml_apply_patchset(xmlNode *xml, const xmlNode *patchset,
                              bool check_version);
 
+void pcmk__patchset_process_digest(const xmlNode *input, xmlNode *result,
```

Comment:
you use source/target below

I'm thinking `pcmk__add_digest_to_patchset`

Reply:
Thanks. I wavered a bit between variable names in new functions... I thought about `before`/`after` as well as `input`/`result`, but ended up sticking with `source`/`target` for now even though it's less intuitive. I can't remember exactly why... I think I ran into some questions about change tracking and the actual content of `target`, and didn't feel like going down that rabbit hole.

And I like `pcmk__add_digest_to_patchset()` better.